tests/common/test_transaction.py

```diff
@@ -1014,16 +989,15 @@ def test_create_transfer_transaction_single_io(tx, user_pub, user2_pub,
     assert transfer_tx.fulfillments_valid([tx.conditions[0]]) is True
 
 
-@mark.skip(reason='FIXME: When divisible assets land')
 def test_create_transfer_transaction_multiple_io(user_pub, user_priv,
```

Comment:
This test was testing to create a `TRANSFER` with multiple inputs and outputs.
Now the `TRANSFER` method just spends the outputs of a single transaction.

Either:

- rename this test's method
- implement multiple IO

Thoughts @r-marques?